stutterhold/Source/PhaseVocodeur/PhaseVocodeur.cpp

```diff
@@ -5,11 +5,28 @@
     Created: 9 Dec 2020 9:19:32pm
     Author:  Julian Vanasse
 
+ By `push`ing into this class, the input is distributed among overlapping buffers,
+ this permits a frequency-domain processing and reconstruction.
+ 
+ Sound comes out via -- TODO: (?)
+ 
+ You can specify your own processing by overwriting the (?) method.
+ 

```

Comment:
Maybe getting a good docstring together would set the tone nicely, help me figure out how we'll use this class.